test/verify/check-machines

```diff
@@ -148,11 +148,19 @@ class TestMachines(MachineCase):
         m.execute("sed -i 's/\"priority\".*$/\"priority\": 100,/' {0}".format("/usr/share/cockpit/machines/manifest.json"))
         m.execute("[ ! -e {0} ] || sed -i 's/\"priority\".*$/\"priority\": 0,/' {0}".format("/usr/share/cockpit/ovirt/manifest.json"))
 
+    def getLibvirtName(self):
+        # Id of service changes in ubuntu distributions
+        # libvirt-bin is primary in ubuntu 1604, lbvirtd is primary in 1710
+        # but there still can be unit-files to both aliases when the service is enabled
+        name = self.machine.execute(
+            "systemctl list-unit-files | grep -o libvirt-bin.service || echo libvirtd.service").replace('\n', '')
+        return self.machine.execute(
+            "systemctl  --property=Id show {0} | cut -c 4-".format(name)).replace('\n', '')
```

Comment:
This should not repeat the same logic. Just special-case `if self.machine.image == "ubuntu-1604":`